src/condor_schedd.V6/qmgmt.h

```diff
@@ -560,7 +573,7 @@ class ClassAdLogTable<JOB_ID_KEY,JobQueueJob*> : public LoggableClassAdTable {
 			Ad = new JobQueueJob(); Ad->Update(*ad); new_ad = true;
 		}
 		Ad->SetDirtyTracking(true);
-		JobQueueJob* payload = reinterpret_cast<JobQueueJob*>(Ad);
+		JobQueueJob payload = reinterpret_cast<JobQueueJob>(Ad);
```

Comment:
This casts a pointer type to an object type.

Reply:
fixed